typescript/src/services/TsParser/CustomCompilerHost.spec.ts

```diff
@@ -22,29 +22,29 @@ describe('CustomCompilerHost', () => {
   describe('getSourceFile', () => {
     it('should return a SourceFile object for a given path', () => {
       const sourceFile = host.getSourceFile('testSrc.ts', 0);
-      expect(sourceFile.fileName).toEqual('testSrc.ts');
-      expect(sourceFile.pos).toEqual(0);
-      expect(sourceFile.text).toEqual(jasmine.any(String));
+      expect(sourceFile!.fileName).toEqual('testSrc.ts');
```

Comment:
nitpick - you could have put the `!` on the end of the call to `getSourceFile` and save adding on each of the following lines.